.github/workflows/iroha2-dev-pr.yml

```
@@ -144,44 +150,67 @@ jobs:
           name: clippy.json
           path: clippy.json
 
-  torii-api-and-client-cli-tests:
+  torii-api-client-cli-docker-compose-tests:
```

```suggestion
  torii-api-cli-tests:
```

no need to specify it's dockerized. People opening PRs don't care

---

Not really, sometimes it is the docker compose launch that crashes and this job should help to prevent it from crashing among other things.